sycl/test-e2e/ClusterLaunch/enqueueLaunchCustom_check_event_deps.cpp

```diff
@@ -0,0 +1,67 @@
+// Checks whether or not event Dependencies are honored by piExtEnqueueLaunchKernelCustom
```

Comment:
Thanks this looks good

Reply:
Another small test reversing the order of these kernels would be great, so you have cluster launch -> normal parallel for dependency

Reply:
Added in [1c35fdd79](https://github.com/intel/llvm/pull/14113/commits/1c35fdd796e87e36b944713bccb25e92d7854f42)